packages/backend/src/dbt/dbtCliClient.ts

```diff
@@ -219,6 +226,12 @@ export class DbtCliClient implements DbtClient {
         );
     }
 
+    static validateSelector(selector: string): boolean {
+        // eslint-disable-next-line no-useless-escape
+        const validSelectorPattern = /^[a-zA-Z0-9\s\-\.\+:_]+$/;
```

Comment:
fyi: Tried passing incorrect value "--target banana --profile banana --project-dir banana" and execa wraps it in a string which is nice and dbt ignored it. 

```
dbt --no-use-colors --log-format json compile --select "--target banana --profile banana --project-dir banana" --profiles-dir "/tmp/local_Bj7kJn" --project-dir "/usr/app/dbt" --target prod --profile lightdash_profile
```